svg_test.go

```diff
@@ -0,0 +1,121 @@
+package jujusvg
+
+import (
+	"bytes"
+
+	gc "gopkg.in/check.v1"
+)
+
+type SVGSuite struct{}
+
+var _ = gc.Suite(&SVGSuite{})
+
+func (s *SVGSuite) TestProcessIcon(c *gc.C) {
+	tests := []struct {
+		about    string
+		icon     []byte
```

Comment:
It's probably easier to define this field and expected as string,
then convert it inside the test.